packages/pkg/src/index.ts

```diff
@@ -13,18 +13,22 @@ import babelParser from 'prettier/plugins/babel.js'
 import { files } from './rules/files.js'
 import { object } from './rules/object.js'
 import { dependencyNames, sort } from './rules/sort.js'
-import type { ObjectExpression, ObjectProperty } from './types.js'
+import type {
+  ObjectExpression,
+  ObjectProperty,
+  FormatOptions,
+} from './types.js'
 
 const PKG_REG = /[/\\]?package\.json$/
 
 const {
   json: { parse },
 } = babelParser.parsers
 
-const format = (properties: ObjectProperty[]) => {
+const format = (properties: ObjectProperty[], options: FormatOptions = {}) => {
```

Comment:
```suggestion
const format = (properties: ObjectProperty[], options: FormatOptions) => {
```

It can't be `undefined` actually according to the codes.

Reply:
nope, when you pass ` = mydefault` to a param, you not only don't need the optional-questionmark for the param, you even CANT use it anymore.

so `, options?: FormatOptions = {}` is invalid TS syntax and raises [ts1015](https://typescript.tv/errors/#ts1015)

Reply:
Now I understand what you mean. You mean because the program logic for calling that part is fully internally ^^'
Yeah okay, I mostly always just design functions like this with an optional options, because I'm used to it, but yeah, will remove it tomorrow.

Reply:
I think we should just pass through the whole options actually.